plugin/Field/Collection/FieldLoaderGeneric.php

```diff
@@ -75,6 +75,7 @@ public function load(): Generator
 	{
 		$newAddressFields = FieldModuleCollectionDecoratorReadAddress::getNewAddressFieldsWithTableNameKey();
 		$result           = $this->sendRequest();
+		$screen           = get_current_screen();
```

Comment:
Why do you need to know the current screen here? I'm unsure what should depend on that.

Reply:
This function is called from the config field of the form, estate and address. So I added "detail estate" screen check so that my new code doesn't affect other config fields causing unexpected errors.

Reply:
Are you sure that this won't lead to issues later on? I find it quite unexpected that the field depends on the page, so this might be confusing later.

What is the worst that could happen if we didn't restrict it to the detail page?

Reply:
It doesn't seem to cause as many errors as I thought, I'll remove it. The code inside the `if screen` I used to move the fields in the `(unnamed category)` to the `Form Specific Fields`, but now it's giving me some errors in the form. I need more time to fix it

![image](https://user-images.githubusercontent.com/101325858/171860630-0c8b2d76-6dc6-4096-b9a4-b7341fd0049d.png)

Reply:
But if you are in a hurry, I think the current code is ok.

Reply:
Let's take the time to get this right. :)

If you find more opportunities to clean up and refactor the code, please do so! ;)

Reply:
I was aiming to do a release next Tuesday, because that is when onOffice enterprise has a release, but there is no real reason why. So let's do the release when this is ready and take the time we need.